packages/graphql-codegen-cli/src/hooks.ts

```diff
@@ -92,7 +94,7 @@ export const lifecycleHooks = (_hooks: Partial<Types.LifecycleHooksDefinition> =
     afterStart: async (): Promise<void> => executeHooks('afterStart', hooks.afterStart),
     onWatchTriggered: async (event: string, path: string): Promise<void> =>
       executeHooks('onWatchTriggered', hooks.onWatchTriggered, [event, path]),
-    onError: async (error: string): Promise<void> => executeHooks('onError', hooks.onError, [`"${error}"`]),
+    onError: async (error: string): Promise<void> => executeHooks('onError', hooks.onError, [error]),
```

Comment:
This change is possibly breaking for `onError` hooks with a custom script as they won't any longer receive quoted text, however, I reckon it's more correct this way, and the impact of this change should be minimal.